types/fundamental/real32quant/README.md

```diff
@@ -0,0 +1,20 @@
+# Real Column Types with variable width
+
+## Fields
+
+ * `FloatReal32Quant2_-1_1`
+ * `DoubleReal32Quant2_-1_1`
+ * `DoubleReal32Quant20_-1_1`
+ * `FloatReal32Quant8_0_1`
+ * `FloatReal32Quant16_0_1`
+ * `DoubleReal32Quant31_-100_25`
+
+with the corresponding field, column types, bit width and value range.
```

Comment:
So to summarize, my suggestion would be the following:

```suggestion
 * `FloatReal32Quant1`
 * `FloatReal32Quant8`
 * `FloatReal32Quant32` with value range `[-pi, pi]`
 * `DoubleReal32Quant1`
 * `DoubleReal32Quant20`
 * `DoubleReal32Quant32` with value range `[-100, 25]`

with the corresponding field types, column type `Real32Quant` with corresponding bit width, and a value range of `[-1, 1]` unless noted otherwise.
```